_posts/2022-04-25-changelog-169.markdown

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #169"
+date: 2022-04-25 13:00:00 +0300
+---
+
+
+## New Features
+
+* [#6836] Initial support for [Run Targets](https://blog.jetbrains.com/idea/2021/01/run-targets-run-and-debug-your-app-in-the-desired-environment/).
+  Run Targets allow you to build/run the applications you’re developing in Docker containers or on remote machines via SSH
+
+  {% include gif-img.html path="/assets/posts/changelog-169/run-target" w="800px" %}
+
+* [#8699] Allow files generated by the `build.rs` buildscripts up to 8 megabytes in size.
+  Previously it was necessary to adjust the limit manually using `idea.max.intellisense.filesize` property.
+  Note, build script evaluation is disabled by default for now. To turn it on, enable `org.rust.cargo.evaluate.build.scripts` [experimental feature][experimental features]
+
+* [#8713] Update [ML completion] model.
+  In addition to the data used before, the new model is now taking into account several new facts to:
+
+  * prefer expressions of type matching the expected type
+
+  * differentiate between inherent, trait and blanket implementations
+
+  * recognise `async`, `const` and `unsafe` contexts
+
+* [#8697] Respect run/debug modes in [Run Anything] by Cargo commands.
+  Now you can run a Cargo command under debugger using <kbd>Shift</kbd> + <kbd>Enter</kbd> in Run Anything popup.
+
+* [#7998] Enable [Valgrind on WSL](https://www.jetbrains.com/help/clion/memory-profiling-with-valgrind.html#valgrind-wsl) toolchains
```

Comment:
Let's be more precise here and use `Valgrind memcheck﻿`.
Also, I think it's worth mentioning that this integration works only in CLion